pkg/volume/flocker/flocker.go

```diff
@@ -293,7 +293,7 @@ func (b *flockerVolumeMounter) SetUpAt(dir string, mounterArgs volume.MounterArg
 
 	datasetState, err := b.flockerClient.GetDatasetState(datasetUUID)
 	if err != nil {
-		return fmt.Errorf("The datasetState for volume with datasetUUID='%s' could not determinted uusing flocker: %s", datasetUUID, err)
+		return fmt.Errorf("The datasetState for volume with datasetUUID='%s' could not determined using flocker: %s", datasetUUID, err)
```

Comment:
Please fix the first character, it should be lowercase. And can you please fix the whole file to lowercase? It would be more useful than fixing one typo.

Reply:
good point! I will do it. Thank you for your suggestion.